src/main/features/response/draft/responseDraft.ts

```diff
@@ -156,6 +161,10 @@ export class ResponseDraft extends DraftDocument {
       && this.partialAdmission !== undefined
   }
 
+  public isResponsePartiallyAdmittedAndAlreadyPaid (): boolean {
```

Comment:
Please make sure you will add unit tests for this method:
test/features/response/draft/responseDraft.ts

Reply:
Done 👍